src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java

```diff
@@ -38,28 +45,45 @@ private boolean isFile(Path file) {
         return !Files.isDirectory(file);
     }
 
-    private boolean isYml(Path file) {
-        return file.toFile().getName().endsWith(".yml");
+    private boolean isYml(Path path) {
+        return Stream
+                .of(YAML_EXT, YML_EXT)
+                .anyMatch(ext -> getFileName(path).endsWith(ext));
     }
 
-    private boolean isNotConfig(Path file) {
-        return !file.toFile().getName().startsWith(".");
+    private boolean isNotConfig(Path path) {
+        return !getFileName(path).startsWith(".");
     }
 
     private Fixture loadFixture(Path file) {
-        String name = getFixtureName(file);
+        val name = getFixtureName(file);
```

Comment:
This change does not concern the feature, so I'd remove it. Besides that, we probably don't have to always use `val`. We can leave simple and short types as they are. `val` is good for hiding long types and types with generics